src/main/java/net/minestom/server/extras/selfmodification/MinestomRootClassLoader.java

```diff
@@ -298,11 +296,7 @@ public boolean loadModifier(URL[] urls, String codeModifierClass) {
             }
             return true;
         } catch (ClassNotFoundException | InvocationTargetException | InstantiationException | IllegalAccessException | NoSuchMethodException e) {
-            if(MinecraftServer.getExceptionManager() != null) {
-                MinecraftServer.getExceptionManager().handleException(e);
-            } else {
-                e.printStackTrace();
-            }
+            MinecraftServer.getExceptionManager().handleException(e);
```

Comment:
I might have missed the change previously, but the exception manager is not guaranteed to be present at this stage. Especially if you are loading a class before `MinecraftServer#init` is called.

Reply:
_Did not notice that_ Ill patch that really quickly then.